app/models/manageiq/providers/ansible_tower/embedded_ansible_worker.rb

```diff
@@ -0,0 +1,20 @@
+class ManageIQ::Providers::AnsibleTower::EmbeddedAnsibleWorker < MiqWorker
```

Comment:
haha, I can see how convinced all of us are to where it belongs.  This is because namespacing it is part of naming it.  And we know how easy naming is...